README.md

```diff
@@ -47,26 +44,25 @@ pnpm stop:synapse
 ### Realm server Hosted App
 In order to run the realm server hosted app:
 
-1. `pnpm start` in the worker/ workspace to build the service worker (you can omit this step if you do not want service worker re-builds).
-2. `pnpm start:build` in the host/ workspace to re-build the host app (this step can be omitted if you do not want host app re-builds)
-3. `pnpm start:base` in the realm-server/ to serve the base realm
-4. `pnpm start:synapse` in the matrix/ workspace to run the matrix server.
+1. `pnpm start:build` in the host/ workspace to re-build the host app (this step can be omitted if you do not want host app re-builds)
+2. `pnpm start:all` in the realm-server/ to serve the base and demo realms
+3. `pnpm start:synapse` in the matrix/ workspace to run the matrix server.
 
-You can visit the URL of each realm server to view that realm's app. So for instance, the base realm's app is available at `http://localhost:4201/base`. Additionally, we have enabled the server that hosts the base realm to also be able to host the local-realm app. To use the local realm visit: `http://localhost:4201/local`.
+You can visit the URL of each realm server to view that realm's app. So for instance, the base realm's app is available at `http://localhost:4201/base` and the demo realm's app is at `https://localhost:4204/`.
 
 Live reloads are not available in this mode, but you can just refresh the page to grab the latest code changes if you are running rebuilds (step #1 and #2 above).
 
 #### Using `start:all`
 
-Instead of running `start:base`, you can alternatively use `pnpm start:all` which also serves a few other realms on other ports--this is convenient if you wish to switch between the app and the tests without having to restart servers. Here's what is spun up with `start:all`:
+WInstead of running `start:base`, you can alternatively use `pnpm start:all` which also serves a few other realms on other ports--this is convenient if you wish to switch between the app and the tests without having to restart servers. Here's what is spun up with `start:all`:
```

Comment:
```suggestion
Instead of running `start:base`, you can alternatively use `pnpm start:all` which also serves a few other realms on other ports--this is convenient if you wish to switch between the app and the tests without having to restart servers. Here's what is spun up with `start:all`:
```